### PR TITLE
[DO NOT MERGE] beta4 api changes

### PR DESCRIPTION
**Pull request by v-dharmarajv**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/communication/Azure.Communication.CallAutomation/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,16 @@
 
 ### Features Added
```
Comment (minwoolee-msft) [inline sdk/communication/Azure.Communication.CallAutomation/CHANGELOG.md:5]: Update the release date

File: sdk/communication/Azure.Communication.CallAutomation/src/CallAutomationClientOptions.cs [hunk 2]
```diff
@@ -52,19 +52,19 @@ public enum ServiceVersion
...
             /// <summary>
-            /// The GA2 (1.1.0) of the CallAutomation service.
+            /// Latest GA3 (1.2.0) of the CallAutomation service.
             /// </summary>
-            V2023_10_15 = 3,
+            V2024_04_15 = 3,
 
             /// <summary>
-            /// Latest ALPHA3 (1.2.0-alpha) preview of the CallAutomation service.
+            /// Latest ALPHA3 (1.3.0-alpha) preview of the CallAutomation service.
```
Comment (minwoolee-msft) [inline sdk/communication/Azure.Communication.CallAutomation/src/CallAutomationClientOptions.cs:65]: wrong summary - should be beta 4

File: sdk/communication/Azure.Communication.CallAutomation/src/CallMedia.cs [hunk 3]
```diff
@@ -1237,5 +1427,132 @@ public virtual async Task<Response> UpdateTranscriptionAsync(string locale, Canc
...
+        public virtual async Task<Response> UpdateTranscriptionAsync(UpdateTranscriptionOptions options = default, CancellationToken cancellationToken = default)
+        {
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(CallMedia)}.{nameof(UpdateTranscription)}");
+            scope.Start();
+            try
+            {
+                UpdateTranscriptionRequestInternal request = new(options.Locale)
+                {
+                    OperationContext = options.OperationContext,
+                    SpeechRecognitionModelEndpointId = options.SpeechRecognitionModelEndpointId
```
Comment (minwoolee-msft) [inline sdk/communication/Azure.Communication.CallAutomation/src/CallMedia.cs:1446]: from GA3, this seems to require input Locale. 
is this part of breaking change that is expected?

Comment (v-vdharmaraj) [reply sdk/communication/Azure.Communication.CallAutomation/src/CallMedia.cs:1446]: We didn't release the transcription in GA3.

File: sdk/communication/Azure.Communication.CallAutomation/src/CallAutomationClientOptions.cs (hunk 2 above)
Comment (v-vdharmaraj) [reply sdk/communication/Azure.Communication.CallAutomation/src/CallAutomationClientOptions.cs]: Updated

File: sdk/communication/Azure.Communication.CallAutomation/CHANGELOG.md (hunk 1 above)
Comment (v-vdharmaraj) [reply sdk/communication/Azure.Communication.CallAutomation/CHANGELOG.md:5]: Updated

**Pull request closed without merging**